scripts/manual_operations/tests/test_util.py

```diff
@@ -0,0 +1,42 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+
+"""
+Test cases for util functions
+"""
+
+from unittest import TestCase
+
+from scripts.manual_operations.util import get_run_range
+
+
+class TestUtil(TestCase):
+    """
+    Test util.py
+    """
+
+    def test_get_run_numbers_one_run_returns_correct_range(self):
+        """
+        Test: Correct range is returned
+        When: second_run is None
+        """
+        self.assertEqual(range(1, 2), get_run_range(1))
+
+    def test_get_run_numbers_valid_input_returns_valid_range(self):
+        """
+        Test: Correct range is returned
+        When: second_run is provided
+        """
+        self.assertEqual(range(1, 5), get_run_range(1, last_run=4))
```

Comment:
To maintain a consistent format when calling a method (`get_run_range()`), we should either call just using raw parameters or with explicit argument assignment, but not interchange between the two e.g

Good practice:
* `my_method(1, 2)`
* `my_method(x=1, y=2)`

Bad practice:
* `my_method(1, y=2)`

Reply:
Interesting, I thought the way I implemented it was idiomatic python, since it shows which param is optional. (Even some of  the examples in pep8 do it) Though I like the consistency. I'll change this and below.

Reply:
@keiranjprice101 @JackEAllen 
The way keiran has called this is actually fine as `last_run` is optional. So the intent is clear. having said that for our own sanctity of mind when testing it's easier to read if it's verbose